src/openforms/contrib/zgw/service.py

```diff
@@ -41,7 +40,7 @@ def create_csv_document(
     name: str,
     csv_data: str,
     options: dict,
-    language: str = "nld",
+    language: str,
```

Comment:
this should use the same type alias (and so the type alias should probably be moved from `documenten.py` into `service.py`), otherwise we lose the benefits of strict types again.

Reply:
So we mean that each function (`create_report_document`, `create_csv_document`, `create_attachment_document`) should have an explicit argument `language` right?In order to define the type in the `service.py` file.

Reply:
Yes, because otherwise you get type check errors - `str` cannot be assigned to the literal :)